### PR TITLE
Grammar, punctuation, and spacing changes to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by arkihtekt**
This change is [Reviewable](https://reviewable.io/reviews/seleniumhq/selenium/846)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -55,11 +52,10 @@ The Selenium project welcomes new contributors. Individuals making
 significant and valuable contributions over time are made _Committers_
 and given commit-access to the project.
 
-If you're looking for easy bugs, have a look at
-[issues labelled E-easy](https://github.com/SeleniumHQ/selenium/issues?q=is%3Aopen+is%3Aissue+label%3AE-easy)
+If you're looking for easy bugs to fix, view issues [marked as E-easy](https://github.com/SeleniumHQ/selenium/issues?q=is%3Aopen+is%3Aissue+label%3AE-easy) 
 on Github.
 
-This document will guide you through the contribution process.
+The following guideline will guide you through the contribution process:
```
Comment (ddavison) [inline CONTRIBUTING.md]: s/guideline/guidelines/

**Comment by ddavison** [comment]
:+1:

**Comment by jparry** [comment]
Done!

File: README.md [hunk 2]
```diff
@@ -49,8 +50,8 @@ To build Selenium, in the same directory as this file:
 
 The order of building modules is determined by the build system.
 If you want to build an individual module
-(assuming all dependent modules have previously been built)
-try something like:
+(assuming all dependent modules have previously been built),
+try the following:
```
Comment (ddavison) [inline README.md]: could you revert this change?  "something like" seems a little better

**Comment by ddavison** [comment]
if you could, squash your commits into one, and implement that one more fix, and we'll be G2G.

**Comment by AutomatedTester** [comment]
Closing as a lot of this has been done on the new Selenium.dev site
